### DISCUSSION TITLE
一键部署vercel  但是一直AI is thinking...

### DISCUSSION THREAD

**Discussion by 18661238005** [ddiu8081/chatgpt-demo#88] [Q&A]
项目克隆了一键部署在vercel 上  网站打开后发起请求一直AI is thinking...    日记里面提示[POST] /api/generate reason=EDGE_FUNCTION_INVOCATION_TIMEOUT, status=504, user_error=true  具体怎么解决！ 大神们帮帮忙 

chatgpt  key  是正常可以使用的。
跪谢了

**Comment by Root-FTW** [comment]
> 项目克隆了一键部署在vercel 上 网站打开后发起请求一直AI is thinking... 日记里面提示[POST] /api/generate reason=EDGE_FUNCTION_INVOCATION_TIMEOUT, status=504, user_error=true 具体怎么解决！ 大神们帮帮忙
> 
> chatgpt key 是正常可以使用的。 跪谢了

Has anyone found a solution to this problem?

An error occurred with your deployment
EDGE_FUNCTION_INVOCATION_TIMEOUT

My subdomain in vercel was working perfect, I didn't modify anything and suddenly it didn't work anymore.

有没有人找到解决这个问题的办法？

我的vercel子域工作得很好，我没有修改任何东西，突然间它就不工作了。

**Comment by yzh990918** [comment]
Ensure that the Vercel environment variable API KEY is valid, Use clashx Pro to enable the enhanced mode and select the US or Japan node.

**Reply by Root-FTW** [reply]
> Ensure that the Vercel environment variable API KEY is valid, Use clashx Pro to enable the enhanced mode and select the US or Japan node.

Does this one work for you? :(

https://chatgpt.ddiu.me/

Thank you for response

**Reply by yzh990918** [reply]
> > Ensure that the Vercel environment variable API KEY is valid, Use clashx Pro to enable the enhanced mode and select the US or Japan node.
> 
> Does this one work for you? :(
> 
> https://chatgpt.ddiu.me/
> 
> Thank you for response

This API overcalled, recommended use your own key.

**Reply by Root-FTW** [reply]
> > > Ensure that the Vercel environment variable API KEY is valid, Use clashx Pro to enable the enhanced mode and select the US or Japan node.
> > 
> > 
> > Does this one work for you? :(
> > https://chatgpt.ddiu.me/
> > Thank you for response
> 
> This API overcalled, recommended use your own key.

I do, look here:
https://github.com/ddiu8081/chatgpt-demo/issues/151

or here (same config):
https://github.com/ddiu8081/chatgpt-demo/issues/153

**Reply by yzh990918** [reply]
> > > > Ensure that the Vercel environment variable API KEY is valid, Use clashx Pro to enable the enhanced mode and select the US or Japan node.
> > > 
> > > 
> > > Does this one work for you? :(
> > > https://chatgpt.ddiu.me/
> > > Thank you for response
> > 
> > 
> > This API overcalled, recommended use your own key.
> 
> I do, look here: #151
> 
> or here (same config): #153

Maybe it's your network problem, this is better to solve it alone, I have no problem deploying it myself.

![image](https://user-images.githubusercontent.com/46418596/223311225-6e0a6d0a-f2da-47d1-8ae7-f85f23a55e24.png)

**Comment by kleenkanteen** [comment]
Same issue happens with me, I am hosting on vercel, error 500 and EDGE_FUNCTION_INVOCATION_TIMEOUT

**Comment by co2color** [comment]
想问问你部署vercel是绑github仓库的项目的吗？我是这样绑的，但github这边只要在.env把apikey写死，然后push代码，我的apikey就会被轮换掉一个新的....请问您怎么处理的？

**Reply by ddiu8081** [reply]
不要把 `.env` 提交到仓库，部署平台一般提供了环境变量功能可以直接设置。
